package_utilis.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+
+def check_import(file_name):
+    """
+    Checks if the filename is a python file and is suitable return True if so.
+    :param file_name: A name of a file
+    :return: True or false
+    :rtype boolean
+    """
+    if file_name == 'modifier.py':
```

Comment:
an explanation for modifier for instance:
in pipeline factory we need to use the Modifier class to get his children.
to do so we need to import modifier.py but importing this with the whole package will raise an error because it will not recognize the Modifier class.
that is why we excluded those files because including them won't give any use